### ISSUE TITLE
[Question]: Preseq module statistics calculation 

### ISSUE THREAD

**Issue by tamuanand** [ewels/MultiQC#2006]
Hi

I am using multiqc 1.15 and I want to know how to use this feature- https://multiqc.info/modules/preseq/#plotting-externally-calculated-read-counts

My multiqc_config.yaml is like this
```yaml
preseq:
  genome_size: hg38_genome
  read_length: 50
  x_axis: counts
  y_axis: coverage
```

I have this file called `STUDY.all_samples_preseq_real_counts_stats.txt` - this was generated using the code snippet given in MultiQC page for the `preseq` module mentioned above

```
Sample_1.preseq.lc_extrap.txt	75154978  75154978
Sample_2.preseq.lc_extrap.txt   79278555  79278555
Sample_3.preseq.lc_extrap.txt   89428837  89428837
```

When I run MultiQC using `--verbose` , I see something like this in the log - I am not sure, how the tool gives out the `Total` and `Unique` from the above data 
- is it a locale issue? I can understand 75154978 becoming 75.154978 but not sure how the Unique is coming as shown below? 
- Does this mean that the MultiQC report for this section is incorrect (I hope not and I hope this is an error in my understanding of how this module should work).

```log
[2023-08-27 10:53:07,891] multiqc.modules.preseq.preseq                      [INFO   ]  Found 3 reports
[2023-08-27 10:53:07,993] multiqc.modules.preseq.preseq                      [DEBUG  ]  Found 3 matching sets of counts from STUDY.all_samples_preseq_real_counts_stats.txt
[2023-08-27 10:53:07,993] multiqc.modules.preseq.preseq                      [DEBUG  ]  Found real counts for Sample_1.preseq.lc_extrap - Total: 75.154978, Unique: 1.2323252714427053
[2023-08-27 10:53:07,993] multiqc.modules.preseq.preseq                      [DEBUG  ]  Found real counts for Sample_2.preseq.lc_extrap - Total: 79.278555, Unique: 1.2999400626524091
[2023-08-27 10:53:07,993] multiqc.modules.preseq.preseq                      [DEBUG  ]  Found real counts for Sample_3.preseq.lc_extrap - Total: 89.428837, Unique: 1.4663754652530194
```

Thanks in advance.

**Comment by tamuanand** [comment]
Hi @vladsavelyev and @ewels 

- Just worth clarifying that MultiQC html is produced but I cannot see where in that html this is applicable  - `To mark on the plot the read counts calculated externally from BAM or fastq files`

Or am I reading too much into those wordings? 

![image](https://github.com/ewels/MultiQC/assets/8467214/9e300919-8f6a-40a4-b17a-3ca1f93d2940)

**Comment by vladsavelyev** [comment]
Hey @tamuanand - thanks for the question! It's been a while since I touched preseq so I was initially puzzled with the numbers as well 😅 

So the numbers in the log message shown are:
* `Total: 75.154978` - that's the number of _millions_ of molecules (similarly to the X axis on the plot)
* `Unique: 1.2323252714427053` - that's those recalculated into coverage, according to the `preseq` section in the config. So it's ~1.23x coverage - similar to the Y axis.

I adjusted now the codebase to make the debug message more clear: https://github.com/ewels/MultiQC/pull/2007
So it would be like this now:
```
Real count for Sample_1.preseq.lc_extrap: 75.1 M (1.23x depth)
```
